docs/conf.py

```diff
@@ -1,3 +1,5 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
```

Comment:
There is some additional cleanup that can happen in this script. I can't make code suggestions from GitHub, but here are the changes:

- Change line 89 to be `html_theme = "sphinx_rtd_theme"`
- Move line 101 (`html_theme_path`) beneath line 89
- Move the `import` on line 98 to the top of the file underneath the copyright header
- Remove line 15: `import os` 
- Remove the `on_rtd` variable from line 94 and the subsequent conditional block beneath it

These changes are required since we don't ever use `readthedocs.org,` which makes the `on_rtd` variable and it's associated conditional logic pointless.